docs/specification/draft/basic/enterprise.mdx

```diff
@@ -0,0 +1,294 @@
+
+If the IdP is multi-tenant and uses the same issuer for all tenants, the MCP Server will already have IdP-specific logic to determine the tenant from the OpenID Connect ID Token (e.g., the `hd` claim for Google) or SAML assertion, and will need to use that if the IdP also has only one client registration for the MCP Server.
+
+`sub` should be an opaque ID, as `iss+sub` is unique. The IdP might want to also include the user's email here, which it should do as a new `email` claim. This would let the app dedupe existing users who may have an account with an email address but have not done SSO yet.
+
+
+## 5. Access Token Request
+
+The MCP Client makes an access token request to the MCP Server's authorization server using the previously obtained Identity Assertion Authorization Grant as a JWT Assertion as defined by [RFC7523](https://datatracker.ietf.org/doc/html/rfc7523).
+
+| Parameter   | Required/Optional | Description                                                                                                        | Example/Allowed Values                                                                           |
+|-------------|-------------------|--------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| `grant_type`| REQUIRED          | MUST be `urn:ietf:params:oauth:grant-type:jwt-bearer`                                                              | `urn:ietf:params:oauth:grant-type:jwt-bearer`<br/>[Section 4.1 of RFC7523](https://datatracker.ietf.org/doc/html/rfc7523#section-4.1) |
+| `assertion` | REQUIRED          | The Identity Assertion Authorization Grant JWT obtained in the previous token exchange step                        | (JWT string)                                                                                     |
+
+The MCP Client authenticates with its credentials as registered with the MCP Server's authorization server.
+
+If the MCP Client has not yet registered with the MCP Server's authorization server, then it can do a Dynamic Client Registration request at this stage.
+
+An example request may look like this:
+
+```
+POST /oauth2/token HTTP/1.1
+Host: acme.chat.example
+Authorization: Basic yZS1yYW5kb20tc2VjcmV0v3JOkF0XG5Qx2
+
+grant_type=urn:ietf:params:oauth:grant-type:jwt-bearer
+assertion=eyJhbGciOiJIUzI1NiIsI...
+```
+
+### 5.1 Processing Rules
+
+All of [Section 5.2 of RFC7521](https://datatracker.ietf.org/doc/html/rfc7521#section-5.2) applies, in addition to the following processing rules:
+
+* Validate the JWT `typ` is `oauth-id-jag+jwt` (per [RFC8725](https://datatracker.ietf.org/doc/html/rfc8725))
+* The `aud` claim **MUST** identify the Issuer URL of the MCP Server's authorization server as the intended audience of the JWT.
+* The `client_id` claim **MUST** identify the same client as the client authentication in the request.
```

Comment:
I think we need something to ground the last sentence about the `client_id` in the real world. "client authentication" is clear to an expert, but not obvious what it means to a casual implementer IMO.

```suggestion
* The `aud` claim of the assertion JWT **MUST** identify the Issuer URL of the MCP Server's authorization server as the intended audience of the JWT.
* The `client_id` claim of the assertion JWT **MUST** identify the same client as the client authentication in the request (e.g., the `client_id` specified in a basic auth header).
```

Reply:
http basic auth is sadly still far more common than private key JWT for OIDC integrations, so I think this is the more helpful example in this case.